crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -67,7 +73,7 @@ fn diagnostic_for_argument(
     replacement: Option<&str>,
 ) -> Option<Diagnostic> {
     let keyword = arguments.find_keyword(deprecated)?;
-    Some(Diagnostic::new(
+    let mut diagnostic = Some(Diagnostic::new(
```

Comment:
Let's not wrap the `Diagnostic` in a `Some` here. That way you don't need to unwrap it with `if let` below. So,
```rust
let keyword = arguments.find_keyword(deprecated)?;
let mut diagnostic = Diagnostic::new(...);
diagnostic.set_fix(...);
Some(diagnostic)
```

Reply:
Thanks, done.